### PR TITLE
[Update Meta.py]

### PR DESCRIPTION
**Pull request by shivanshu1086**
Solved issue #32 , without removing other fields, You can still able to push the data to the database without the design field or if you want to add something in it you can do that too...
Screenshot 2020-03-11 at 12 44 01 PM

### CODE REVIEW

**Comment by dlebauer** [comment]
This PR appears to add a new endpoint /cultivars_whitespace_test that I think is independent of #32 which is to enable the /experiments endpoint to accept csv files that optionally contain a 'design' field. Perhaps this was caused by merging this branch with one related to #31 

But I also don't think that this endpoint would be necessary for #31 - stripping whitespace should be done transparently without any changes to the api definition other than documenting the feature. But as discussed in #31 stripping whitespace might be more effectively handled by the database as a trigger function.

**Comment by shivanshu1086** [comment]
Hey @dlebauer , It happens because of merging the branches...I have created two separate PR's for those.
But adding the new endpoint /cultivars_whitespace_test will be helpful for the issue #31 to test that out.
but you can see the changes in Meta.py for #32 . Now you can add the design or not it's not necessary.

File: app/yaba.yaml [hunk 1]
```diff
@@ -737,7 +737,82 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /cultivars_whitespace_test:
```
Comment (Chris-Schnaufer) [inline app/yaba.yaml:740]: Should this be removed from the commit since it's test-only; don't want end users to do this

**Comment by dlebauer** [comment]
@shivanshu1086 so is line 66 of Meta.py the only line required to make design optional? https://github.com/PecanProject/BETYdb-YABA/pull/37/files#diff-29fb4bd353198f4e1741cdd3ab2689c3R66

It would be great to have tests in this repository, but I am still concerned about adding new endpoints dedicated to tests - is this apparoach commonly used?

**Comment by dlebauer** [comment]
Thanks for the contribution @shivanshu1086 - closing now as this was fixed in pr #35

**Comment by dlebauer** [comment]
reopening because this wasn't duplicated by #35; this still has the part that strips whitespace

**Pull request closed without merging**